src/test/java/bio/terra/service/snapshotbuilder/utils/constants/Observation.java

```diff
@@ -0,0 +1,8 @@
+package bio.terra.service.snapshotbuilder.utils.constants;
+
+public final class Observation {
+  private Observation() {}
+
+  public static final String TABLE_NAME = "observation";
```

Comment:
Not that it really matters because it is test data but I noticed there is no observation occurrence table, just observation table in OMOP so I renamed this class